modules/virt-hot-plugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -6,16 +6,16 @@
 [id="virt-hot-plugging-a-virtual-disk-using-the-web-console{context}"]
 = Hot-plugging a virtual disk using the web console
 
-Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running. When you hot-plug a virtual disk, it remains attached to the VMI until you hot-unplug it.
 
 .Prerequisites
 * You must have a running virtual machine to hot-plug a virtual disk.
 
 .Procedure
 
-. Click *Virtualization* -> *VirtualMachines* from the side menu.
+. Click *Virtualization* from the side menu.
 
-. Select a running virtual machine to open the *VirtualMachine details* page.
+. On the *Virtual Machines* tab, select the running virtual machine to which you want to hot-plug a virtual disk.
```

Comment:
Based on the UI, I would suggest rephrasing this as: `Select the running virtual machine to which you want to hot-plug a virtual disk. `

Reply:
Done.